tiatoolbox/tools/stainextract.py

```diff
@@ -79,10 +79,12 @@ class CustomExtractor:
 
     """
 
-    def __init__(self, stain_matrix):
+    def __init__(self, stain_matrix) -> None:
+        """Initializes :class:`CustomExtractor`."""
```

Comment:
```suggestion
        """Initialize :class:`CustomExtractor`."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/c2e1cd4ccfd6aaeb32664f23d4f6634cddb90090